### PR TITLE
Off canvas tray bug fix

### LINKED ISSUE #7115: Off-canvas tray default width is too wide in Safari and Firefox
# Problem
Off-canvas tray default width is too wide in Safari and Firefox

# Steps to reproduce
Edit https://education.uiowa.edu/node/5076/layout events block in Safari or Firefox

Screenshot 2023-11-14 at 10 27 25 AM

# Proposed solution

- There is a console error when the tray is first opened. Might want to start there. 
- Review Dustin's comments below to see if they point us in a direction https://github.com/uiowa/uiowa/issues/7115#issuecomment-1811443518.
- Potentially investigate cookies js to see if that could be causing any errors. 
- Check handle offscreen prevention calculation maybe.

### PR DESCRIPTION
**Pull request by e-marie-w**
Resolves #7115.

# How to test
- checkout branch
- `ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli`
- Configure a layout builder block on an existing page or a add a new page and block and configure that
- See that the off-canvas tray for the block no longer takes up the entire screen. 
- Bonus: inspect the page and click the application tab. You should see a `ui_off_canvas_width` cookie with a value of 500
- You can also try adjusting the tray width, ~~but for me the change is only saved some of the time.~~ because of the patch the tray width is now saved correctly and actually updates as expected!

### CODE REVIEW

File: docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js [hunk 1]
```diff
@@ -26,11 +26,12 @@
 
           let justCreated = true;
 
-          if (Drupal.offCanvas.isOffCanvas($element) && $element.find('.layout-selection').length === 1) {
+          // Removed $element.find('.layout-selection').length check.
```
Comment (pyrello) [inline docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js]: I think a comment would be more useful here if it explained what is happening (if its not obvious). The relevance of this particular comment will be gone pretty much as soon as this gets merged.

**Comment by pyrello** [comment]
Pulling to test.

**Comment by e-marie-w** [comment]
I finally had a chance to check the patch, it works great!!

**Review by pyrello — APPROVED** [review]
This is an improvement no doubt.

**Pull request merged**